### PR TITLE
Utilize DiskStorage

### PR DESCRIPTION
**Pull request by topocount**
This CLI refactor is effectively a no-op. The purpose here was to
replace any filesystem I/O that directly occurs in any CLI command file.
Eventually, this will be refactored again to enable the compression of
instance-specific files such as CredGraph data.

Builds towards: #2768 
merge after #2805  
merge next: #2840 

# Test Plan
1. yarn && yarn build
1. cd sharness/**snapshots**/test-instance
1. scdev go
1. scdev serve

Verify that all data loads successfully in the webpage.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200032326400385/1200038249441670) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: src/cli/score.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // @flow
 
-import fs from "fs-extra";
 import {join as pathJoin} from "path";
 import stringify from "json-stable-stringify";
```
Comment (blueridger) [inline src/cli/score.js:4]: You can actually skip this file if you want.

**Review by blueridger — APPROVED** [review]

File: src/api/instance/localInstance.js [hunk 2]
```diff
@@ -169,24 +180,21 @@ export class LocalInstance implements Instance {
...
+    const ledgerPath = pathJoin(...LEDGER_PATH);
+    await this._storage.set(ledgerPath, encode(ledger.serialize()));
   }
 
   async writeCredGraph(credGraph: CredGraph): Promise<void> {
     const cgJson = stringify(credGraph.toJSON());
-    const outputPath = pathJoin(this._baseDirectory, ...CREDGRAPH_PATH);
-    await fs.writeFile(outputPath, cgJson);
+    const outputPath = pathJoin(...CREDGRAPH_PATH);
+    await this._storage.set(outputPath, encode(cgJson));
```
[the remark this answers is not in the document]
Comment (blueridger) [reply src/api/instance/localInstance.js:190]: ah I see. For composition with ZipStorage.

File: src/api/instance/localInstance.js [hunk 3]
```diff
@@ -54,9 +56,11 @@ on the local disk.
  */
 export class LocalInstance implements Instance {
   _baseDirectory: string;
+  _storage: DiskStorage;
 
   constructor(baseDirectory: string) {
     this._baseDirectory = baseDirectory;
```
[the remark this answers is not in the document]
Comment (topocount) [reply src/api/instance/localInstance.js:62]: :+1: will remove

[the remark this answers is not in the document]
Comment (topocount) [reply src/api/instance/localInstance.js:62]: update: looked into removing this, and it's needed to scaffold plugin directories.

**Pull request merged**